api/test-context/src/test_context.rs

```diff
@@ -310,6 +312,22 @@ impl TestContext {
         LocalAccount::generate(self.rng())
     }
 
+    pub async fn create_account(&mut self, root: &mut LocalAccount) -> LocalAccount {
```

Comment:
I've thought about that. but in [test_resource_group](https://github.com/aptos-labs/aptos-core/blob/d4fe07f34cc4fedddaf59107c337406239d65e99/api/src/tests/resource_groups.rs#L27-29) we need to use the same root account to create three accounts accordingly. If I use [self.root_account()](https://github.com/aptos-labs/aptos-core/blob/d4fe07f34cc4fedddaf59107c337406239d65e99/api/test-context/src/test_context.rs#L302), it will create a new root account with the same address and private key. That makes the test failed.

Reply:
You can refactor root_account to make sure that doesn't happen. I actually did this in my multisig account PR but that has not been merged yet: https://github.com/aptos-labs/aptos-core/pull/5894/files#diff-4793371cd68354cfb0b1e1824c99962391396eea99766757e70ffd003b654007. You can make those changes here.